photon-core/src/main/java/org/photonvision/vision/camera/QuirkyCamera.java

```diff
@@ -81,10 +83,9 @@ public class QuirkyCamera {
             new QuirkyCamera(
                     -1,
                     -1,
-                    "mmal service 16.1",
-                    CameraQuirk.PiCam,
+                    "mmal service 16.1", // TODO - is this still accurate in the libpicam days?
```

Comment:
I think this should no longer show up as mmal service on any supported platform, not sure.

Reply:
Early enough in summer to rip it out and let it go to the scream test?